priv/static/js/app.js

```diff
@@ -1,2 +1,2 @@
._timer=r?setInterval(this._flushToChart.bind(this),+n.refreshInterval):null}return rt(e,[{key:"clearTimers",value:function(){clearInterval(this._timer)}},{key:"resize",value:function(e){this.uplotChart.setSize({width:Math.max(e.width,lt),height:ht})}},{key:"pushData",value:function(e){e.length&&(this._isBufferingData?this._pushToBuffer:this._pushToChart).call(this,e)}},{key:"_pushToBuffer",value:function(e){this._buffer=this._buffer.concat(e)}},{key:"_pushToChart",value:function(e){this.metric.handleMeasurements(e)}},{key:"_flushToChart",value:function(){var e=this._flushBuffer();e.length&&this._pushToChart(e)}},{key:"_flushBuffer",value:function(){if(this._buffer&&!this._buffer.length)return[];var e=this._buffer;return this._buffer=[],e.reduce((function(e,t){return e.concat(t)}),[])}}]),e}(),kt={PhxChartComponent:{mounted:function(){var e,t,n,i=this,r=this.el.parentElement.querySelector(".chart"),o=r.getBoundingClientRect(),a=Object.assign({},r.dataset,{tagged:r.dataset.tags&&""!==r.dataset.tags||!1,width:Math.max(o.width,lt),height:ht,now:new Date/1e3,refreshInterval:1e3});this.chart=new bt(r,a),window.addEventListener("resize",(e=function(){var e=r.getBoundingClientRect();i.chart.resize(e)},n=!1,function(){n||(requestAnimationFrame(e),n=!0,setTimeout((function(){n=!1}),t))}))},updated:function(){var e=Array.from(this.el.children||[]).map((function(e){var t=e.dataset;return{x:t.x,y:+t.y,z:+t.z/1e6}}));e.length>0&&this.chart.pushData(e)},destroyed:function(){this.chart.clearTimers()}},PhxRequestLoggerCookie:{updated:function(){var e,t,n={key:(e=this).el.getAttribute("data-cookie-key"),value:e.el.getAttribute("data-cookie-value")};t=n,document.cookie="".concat(t.key,"=; expires=").concat("Thu, 01 Jan 1970 00:00:00 GMT"),function(e){return"true"===e.el.getAttribute("data-cookie-enabled")}(this)&&function(e){document.cookie="".concat(e.key,"=").concat(e.value,";samesite=strict;path=/")}(n)}},PhxRequestLoggerQueryParameter:{mounted:function(){var e=this;this.el.querySelector(".btn-primary").addEventListener("click",(function(t){!function(e){if(navigator.clipboard){var t=e.value;navigator.clipboard.writeText(t)}else e.select(),e.setSelectionRange(0,99999),document.execCommand("copy")}(e.el.querySelector("textarea"));var n=e.el.querySelector(".copy-indicator");n.setAttribute("data-enabled","false"),n.offsetWidth,n.setAttribute("data-enabled","true")}))}},PhxRequestLoggerMessages:{updated:function(){if(this.el.querySelector(".logger-autoscroll-checkbox").checked){var e=this.el.querySelector("#logger-messages");e.scrollTop=e.scrollHeight}}}},wt=document.querySelector("html").getAttribute("phx-socket")||"/live",xt=document.querySelector("meta[name='csrf-token']").getAttribute("content"),St=new a.LiveSocket(wt,i.Socket,{hooks:kt,params:{_csrf_token:xt}});window.addEventListener("phx:page-loading-start",(function(e){return o.a.start()})),window.addEventListener("phx:page-loading-stop",(function(e){return o.a.done()})),St.connect(),window.liveSocket=St}]);
```

Comment:
Please revert these changes too.

Reply:
See above, assume same resolution whatever we decide there